standard/cli/command-reference/porter-job.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: 'porter job'
+---
+
+##### Prerequisites
+
+- You've logged in to the Porter CLI after running [porter auth login](/enterprise/cli/command-reference/porter-auth)
+- You're connected to the correct project by running [porter config set-project](/enterprise/cli/command-reference/porter-config)
+
+### `porter job runs`[](#porter-job-runs 'Direct link to heading')
+
+The `porter job runs` command allows users to view the job runs for a project, with optional filters applied. The returned job runs will be sorted by creation time, with the most recent job runs appearing first.
+
+```
+porter job runs [JOB_NAME]
+```
+
+The following flags can be used to filter the job runs:
+
+```yaml
+# --job:
+# filter job runs by job name
+porter job runs --job [JOB_NAME]
+
+# --app:
+# filter job runs by application name
+porter job runs --app [APP_NAME]
+
+# --revision:
+# filter job runs by revision number (requires --app)
+porter job runs --app [APP_NAME] --revision [REVISION_NUMBER]
+
+# --target:
+# filter job runs by deployment target, will use the default target if not provided
+# can pass "ALL" to pull runs across all deployment targets
```

Comment:
Minor comment but I think having a different flag `--all-targets` may be more useful here